nrf_802154/doc/rd_service_layer_lib.rst

```diff
@@ -1,7 +1,7 @@
 .. _nrf_802154_sl:
 
 nRF 802.15.4 Service Layer
-##########################
+==========================
```

Comment:
@e-rk We should keep the sdk-nrfxlib's doc version as is and backport the change to internal repo.
See https://github.com/nrfconnect/sdk-nrfxlib/commit/390e818f6ddf400b78862efc2205ecc59191055d

Reply:
Removed this change.